go.mod

```diff
@@ -2,8 +2,10 @@ module decred.org/dcrdex
 
 go 1.15
 
+replace decred.org/dcrwallet => github.com/chappjc/dcrwallet v1.6.3-0.20210324151833-873c564976fd
```

Comment:
Temporary, of course.  The dcrwallet require will be updated when/if https://github.com/decred/dcrwallet/pull/2020 is merged and backported to the dcrwallet release-v1.6 branch.

Reply:
The replace is now removed, and the dcrwallet require updated.